plugin/skills/agentmemory-config/SKILL.md

```
@@ -0,0 +1,37 @@
+name: agentmemory-config
+description: agentmemory configuration, environment variables, ports, and feature flags. Use when enabling a feature, changing ports, setting an API key, configuring auth, or explaining why a feature is off by default.
+user-invocable: false
+---
+
+agentmemory reads configuration from the environment and from `~/.agentmemory/.env` (one `KEY=value` per line, no `export` prefix). Restart the server after changing it.
+
+## Quick start
+
+Enable richer memory and set a provider key in `~/.agentmemory/.env`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)